cli/mender-artifact/validate_test.go

```diff
@@ -66,7 +66,6 @@ var validateTests = []struct {
 	validateKey   []byte
 	expectedError error
 }{
-	{1, nil, nil, nil},
```

Comment:
This is still a valid test case. Upgrade it to v2 instead of removing it

Reply:
This might apply to other places as well. I have not reviewed your changes, but while working on [other issue](https://tracker.mender.io/browse/MEN-2802) it occurred to me that you might be just removing this test case.

Reply:
How is it still a valid test? It has no key. 
I figured this was its V2 equivalent:
`	{2, []byte(PrivateValidateRSAKey), nil, nil}`

Reply:
That is exactly the test case I am fixing. See https://github.com/mendersoftware/mender-artifact/pull/227